documentation.md

```diff
@@ -146,3 +146,25 @@ organization = github.Organization(organization_name="Clueless-Community")
 | `.repository_details()`     | Returns the list of repositories with their details.            |
 | `.pinned_repository()`      | Returns the list of pinned repositories with their details.     |
 | `.get_organization_links()` | Returns a dictionary of important website links of a community. |
+
```

Comment:
If you have noticed, we added the endpoints in the `dev-documentation. md` file, there you have to maintain the documentation. So update it accordingly.

Reply:
done.